.github/workflows/gradle.yml

```diff
@@ -41,11 +41,19 @@ jobs:
       - name: Ensure no file change
         run: git diff-index --quiet HEAD
 
-      - name: Build Docker Images (Master branch only)
+      - name: Build Core Docker Images (Master branch only)
         if: success() && github.ref == 'refs/heads/master'
         run: docker-compose -f docker-compose.build.yaml build
```

Comment:
is this needed for integration tests? if not, I'd move it after integration tests since it takes a long time to run

Reply:
Good point.